tensorboard/plugins/debugger/tf_debugger_dashboard/tf-tensor-value-view.html

```diff
@@ -246,13 +241,12 @@
           this.set('_dataScalar', tensorData);
         } else if (tensorRankWithTime === 1) {
           this._setVisualizationType('lineChart');
-          let xData = [];
+          let xyData = {'x': [], 'y': tensorData};
           // TODO(cais): For history mode, this should be time steps.
           for (let i = 0; i < tensorData.length; ++i) {
-            xData.push(i + 1);
+            xyData['x'].push(i + 1);
```

Comment:
Why +1?

Reply:
Just to let the x axis start from 1 (be 1-based), which I think is more intuitive than 0-based.